src/main/webapp/dataverse-textMessages.xhtml

```diff
@@ -17,12 +17,15 @@
         <ui:define name="body">
             <f:metadata>
                 <f:viewParam name="dataverseId" value="#{TextMessagePage.dataverseId}"/>
+                <f:viewParam name="dataverseId" value="#{BannerTab.dataverseId}"/>
```

Comment:
Yeah, i was wondering about it, and I went for Tab because usually page's have their own xhtml file and tab is just a separate logic on the same page.